app/contacts/contact_tcp.py

```diff
@@ -20,7 +19,7 @@ def __init__(self, services):
     async def start(self):
         loop = asyncio.get_event_loop()
         tcp = self.get_config('app.contact.tcp')
-        loop.create_task(asyncio.start_server(self.tcp_handler.accept, '0.0.0.0', tcp.split(':')[1], loop=loop))
+        loop.create_task(asyncio.start_server(self.tcp_handler.accept, '127.0.0.1', tcp.split(':')[1], loop=loop))
```

Comment:
actually... we can't make this change.. we need to serve the port on all interfaces so external agents can connect.. can you find a different approach to work?

Reply:
if you swap these back to 0.0.0.0 (in all 3 files) i'll merge the other changes-- as they do fix a bug

Reply:
Will do.  Is Caldera platform-aware?  Could it set variables based on the platform it is running on?